bash_scripts/batch_atm.NERSC.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -l
+#SBATCH --partition=regular
```

Comment:
@salilmahajan: what is your opinion on this? Have you ever run the run_EDISON.sl batch script? That was what I took as an example to make `batch_atm.edison.bash`.